handler/instance_started.go

```diff
@@ -212,15 +199,15 @@ func (h *InstanceStarted) getCodeListsFromInstance(i *recipe.Instance) ([]string
 	return codelists, nil
 }
 
-func (h *InstanceStarted) createUpdateInstanceRequest(ctx context.Context, mf gql.Variables, e *event.InstanceStarted, ctblrBlob string, codelists []recipe.CodeList) dataset.UpdateInstance {
+func (h *InstanceStarted) CreateUpdateInstanceRequest(ctx context.Context, format *string, mf gql.Variables, e *event.InstanceStarted, ctblrBlob string, codelists []recipe.CodeList, edition string) dataset.UpdateInstance {
 	req := dataset.UpdateInstance{
-		Edition:    "2021",
+		Edition:    edition,
 		CSVHeader:  []string{cantabularTable},
 		InstanceID: e.InstanceID,
-		Type:       cantabularTable,
+		Type:       *format,
```

Comment:
A pointer to a string only makes sense if we expect at some point the value to be `nil`.
Could you explain the logic behind this? :)